server/routes/bookedtoday/unexpectedArrivals/index.ts

```diff
@@ -1,70 +1,84 @@
 import express, { RequestHandler, Router } from 'express'
-import SearchForExistingRecordController from './searchForExistingRecordsController'
-import NoExistingRecordsFoundController from './noExistingRecordsFoundController'
-import SingleExistingRecordFoundController from './singleExistingRecordFoundController'
-import MultipleExistingRecordsFoundController from './multipleExistingRecordsFoundController'
-import authorisationForUrlMiddleware from '../../../middleware/authorisationForUrlMiddleware'
+import AddPersonalDetailsController from './addPersonalDetailsController'
 import asyncMiddleware from '../../../middleware/asyncMiddleware'
-import { Services } from '../../../services'
+import authorisationForUrlMiddleware from '../../../middleware/authorisationForUrlMiddleware'
+import DateOfBirthValidation from '../arrivals/validation/dateOfBirthValidation'
+import MatchedRecordSelectionValidation from '../arrivals/searchforexisting/validation/matchedRecordSelectionValidation'
+import MultipleExistingRecordsFoundController from './multipleExistingRecordsFoundController'
+import NameValidation from '../arrivals/validation/nameValidation'
+import NoExistingRecordsFoundController from './noExistingRecordsFoundController'
 import Role from '../../../authentication/role'
-import validationMiddleware from '../../../middleware/validationMiddleware'
-import SearchForExistingRecordsValidation from './validation/searchForExistingRecordsValidation'
-import DateOfBirthValidation from './validation/dateOfBirthValidation'
-import PrisonNumberValidation from './validation/prisonNumberValidation'
 import PncNumberValidation from './validation/pncNumberValidation'
-import MatchedRecordSelectionValidation from '../arrivals/searchforexisting/validation/matchedRecordSelectionValidation'
+import PrisonNumberValidation from './validation/prisonNumberValidation'
+import SearchForExistingRecordController from './searchForExistingRecordsController'
+import SearchForExistingRecordsDateOfBirthValidation from './validation/dateOfBirthValidation'
+import SearchForExistingRecordsValidation from './validation/searchForExistingRecordsValidation'
+import { Services } from '../../../services'
+import SexValidation from './validation/sexValidation'
+import SingleExistingRecordFoundController from './singleExistingRecordFoundController'
+import validationMiddleware from '../../../middleware/validationMiddleware'
 
 export default function routes(services: Services): Router {
   const router = express.Router()
+  const search = '/search-for-existing-record'
```

Comment:
the new 'add-personal-details' url does not contain the 'search-for-existing-record' segment so this has been extracted to a constant and only passed to all but the 'add-personal-details' endpoint